new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -0,0 +1,309 @@
+import static oracle.weblogic.kubernetes.utils.CommonUtils.createMiiDomain;
+import static oracle.weblogic.kubernetes.utils.CommonUtils.installAndVerifyNginx;
+import static oracle.weblogic.kubernetes.utils.CommonUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.TestUtils.callWebAppAndCheckForServerNameInResponse;
+import static oracle.weblogic.kubernetes.utils.TestUtils.getNextFreePort;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+/**
+ * Simple JUnit test file used for testing operator usability.
+ * Use Helm chart to install operator(s)
+ */
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test operator usability using Helm chart installation")
+@IntegrationTest
+class ItUsabilityOperatorHelmChart implements LoggedTest {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static HelmParams nginxHelmParams = null;
+  private static int nodeportshttp = 0;
+
+  // domain constants
+  private final String domainUid = "domain1";
+  private final String clusterName = "cluster-1";
+  private final int managedServerPort = 8001;
+  private final int replicaCount = 2;
+  private final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+
+  /**
+   * Get namespaces for operator, domain1 and NGINX.
+   * Install and verify NGINX.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // get a unique NGINX namespace
+    logger.info("Getting a unique namespace for NGINX");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    String nginxNamespace = namespaces.get(2);
+
+    // get a free node port for NGINX
+    nodeportshttp = getNextFreePort(30305, 30405);
+    int nodeportshttps = getNextFreePort(30443, 30543);
+
+    // install and verify NGINX
+    logger.info("Installing and verifying NGINX");
+    nginxHelmParams = installAndVerifyNginx(nginxNamespace, nodeportshttp, nodeportshttps);
+  }
+
+  /**
+   * Create operator and verify it is deployed successfully.
+   * Create domain1 and verify all server pods in domain1 were created and ready.
```

Comment:
"Create domain1"   does not make any sense. Better say "Create a Custom Domain Resource with DomanUid as domain1"  Here domain1 is a string, not a K8s Object.

Reply:
fixed